cli/Application/Command/Make/Langtemplates.php

```diff
@@ -73,7 +73,7 @@ public function execute()
 		$this->getApplication()->outputTitle('Make Language templates');
 
 		ExtensionHelper::addDomainPath('Core', JPATH_ROOT . '/src');
-		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media');
+		ExtensionHelper::addDomainPath('CoreJS', JPATH_ROOT . '/www/media/js');
 		ExtensionHelper::addDomainPath('Template', JPATH_ROOT . '/cache/twig');
```

Comment:
Hmm I'm not sure. The Twig path hasn't been changed by this PR, so everything should work.

The Javascript path on the other hand has changed (along with some "excludes" because it failed to run on my system and also hasn't changed since march (when somebody merged #759 claiming he tested it...) Compare the `POT-Creation-Date` on 
https://github.com/joomla/jissues/blob/master/src/JTracker/g11n/templates/JTracker.pot#L11 
and 
https://github.com/joomla/jissues/blob/master/src/JTracker/g11n/templates/JTracker.js.pot#L11

I admit that this "fix" shouldn't have been included in this PR...